Source/Core/DolphinWX/Debugger/CodeWindow.h

```diff
@@ -128,6 +123,9 @@ class CCodeWindow : public wxPanel
   int iNbAffiliation[IDM_DEBUG_WINDOW_LIST_END - IDM_DEBUG_WINDOW_LIST_START];
 
 private:
+  // Parent interaction
```

Comment:
I know you just moved the comment from the `public` section, but as it stands it barely helps (and would likely benefit more from being renamed to `GetParentMenuBar`, `GetOwningMenuBar` or similar).